includes/definitions/coriant.yaml

```diff
@@ -0,0 +1,47 @@
+os: coriant
+text: 'Coriant TNMS'
+type: network
+icon: coriant
+discovery_modules:
+    ntp: 0
+    ospf: 0
+    stp: 0
+    ucd-diskio: 0
+    ucd-mib: 0
+    ipmi: 0
+    bgp-peers: 0
+    toner: 0
+    services: 0
+    loadbalancer: 0
+    libvirt-vminfo: 0
+    vmware-vminfo: 0
+    vlans: 0
+    junose-atm-vp: 0
+    arp-table: 0
+    mef-evc: 1
+    cisco-vrf-lite: 0
+    storage: 0
+poller_modules:
```

Comment:
From poller_modules you can remove the following as they are all 0 by default:

junose-atm-vp
toner
loadbalancers